frontend/src/app/card.css

```diff
@@ -0,0 +1,14 @@
+.card {
```

Comment:
Conflicts with bootstrap. Event form no longer fills width when on desktop and browser is narrow. 
![screen shot 2018-04-06 at 11 00 30 am](https://user-images.githubusercontent.com/36180018/38428501-e5c47d6c-3989-11e8-98a9-67b86cee2d33.png)